cashu/wallet/__init__.py

```diff
@@ -1,3 +1,5 @@
 import sys
+import pkg_resources
 
 sys.tracebacklimit = None  # type: ignore
+__version__ = pkg_resources.get_distribution("cashu").version
```

Comment:
this though requires that the package `cashu` is installed, ie a `poetry install --no-root` will break it

Reply:
I don't get this. Why is this here?

Reply:
this is "just" for:
```python
>>>from cashu import wallet
>>>wallet.__version__
'0.5.1'
```
I can remove it if you feel it's unnecessary.